window.py

```diff
@@ -594,8 +594,6 @@ def start():
     app.installTranslator(translator)
     app.installTranslator(bt)
 
-    bt.loadCfgTranslation()
```

Comment:
In `gui/util/translator.py`, I updated the code to load the config translation in the `__init__` method. Since this doesn’t appear to be a time-consuming task, there’s no need to delay the loading process by a manual call. Manually initializing processes can be error-prone and should be avoided when possible.

Reply:
Oh, I just mean you may test whether some side effects exist, and your code may not necessarily be wrong.

Reply:
I figure out. When the bt (Translator) [initialization code ](https://github.com/pur1fying/blue_archive_auto_script/commit/019f8b3a86cc5a8a56b7000636680d7884eb085d#diff-8933e7b9c575f34d9765c2f676ac2f24940fd2c79a117886e397ef74f0184ac5R410) was written, it need the locale from config which only available in `window.py`, so the initialization process must happen in `window.py`. But now since the locale already available in `translator.py`, we can safely do the initialization process in `__init__` method.

I've verified the result with GUI, which works perfectly.

Reply:
Ok, perfect for this change.